crates/core_simd/src/vector.rs

```diff
@@ -3,70 +3,92 @@ use crate::simd::{
     SimdPartialOrd, SupportedLaneCount, Swizzle,
 };
 
-/// A SIMD vector of `LANES` elements of type `T`. `Simd<T, N>` has the same shape as [`[T; N]`](array), but operates like `T`.
+/// A SIMD vector with the shape of `[T; VL]` but the operations of `T`.
```

Comment:
While I understand the choice of `VL`, I think it's still slightly jargon-ish. I would prefer `N` because it doesn't mean anything, just as `[T; AL]` would be slightly unnecessary.

Reply:
I'm not overly attached to it, no. It does make me wonder why the `std` doesn't refer to arrays as `[T; L]` in general, though.

Reply:
I think it's helpful that one might see `N` and just think "number". And at worst, it's just another letter

Reply:
I think it's because one commonly talks of "n elements" and "O(n)" and such, and nobody ever wants to say "O(l)" because fonts, so avoids `L` too.